.github/workflows/rentzila_tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Rentzila Playwright Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: macos-latest
```

Comment:
Why do you run on macos and not ubuntu?

Reply:
Because I locally run tests on Mac. In some tests, I imitate copy-paste value into an input using Mac keyboard shortcut